cfg/config.go

```diff
@@ -50,7 +57,28 @@ func GetConfig() (*Config, error) {
 			config.PushTimeout = time.Duration(pushTimeout) * time.Second
 		}
 	} else {
-		config.PushTimeout = 10 * time.Second
+		config.PushTimeout = defaultPushTimeout
+	}
+
+	bs := os.Getenv("ED_BATCH_SIZE")
```

Comment:
can you update the README for this new env variable and also `template.yaml.tmpl` file

Reply:
done